data/presets/office/lawyer/notary.json

```diff
@@ -0,0 +1,34 @@
+{
+    "icon": "fas-stamp",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "office": "lawyer",
+        "lawyer": "notary"
+    },
+    "reference": {
+        "key": "office",
+        "value": "notary"
+    },
+    "locationSet": {
+        "exclude": [
+            "US",
+            "UK",
+            "CA",
+            "AU",
+            "IE",
+            "NZ"
+        ]
```

Comment:
I think this has it backwards: non-lawyer notaries in civil law countries but lawyer notaries in common law countries?

Reply:
Oh, I see, this preset only exists to have a more generic name in civil law countries, where “notary” is assumed to mean “civil law notary”.